src/allmydata/test/test_testing.py

```diff
@@ -0,0 +1,38 @@
+from twisted.trial.unittest import (
+    TestCase,
+)
+from twisted.internet.defer import (
+    inlineCallbacks,
+)
+
+from allmydata.testing.web import (
+    create_tahoe_treq_client,
+)
+
+
+class FakeWebTest(TestCase):
+    """
+    Test the WebUI verified-fakes infrastucture
+    """
+
+    def setUp(self):
+        super(FakeWebTest, self).setUp()
+        self.http_client = create_tahoe_treq_client()
+
+    @inlineCallbacks
+    def test_create_and_download(self):
+        """
+        Upload some content and then download it
```

Comment:
It might be nice to mention the endpoint being tested here.  `PUT /uri` I think?  Though also `GET /uri?uri=CAP`.